package.json

```diff
@@ -6,28 +6,37 @@
   "private": true,
   "scripts": {
     "build": "rollup -c",
-    "commit": "git-cz"
+    "commit": "git-cz",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "test:ci": "jest --ci",
+    "lint": "foundry run eslint . --ext .js,.jsx,.json,.ts,.tsx",
+    "lint:fix": "yarn lint --fix",
+    "lint:ci": "yarn lint --format junit -o __reports__/eslint-results.xml"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^1.12.0",
-    "@typescript-eslint/parser": "^1.12.0",
+    "@sumup/foundry": "^3.1.1",
+    "@types/jest": "^25.1.3",
+    "@types/tti-polyfill": "^0.2.0",
+    "@typescript-eslint/eslint-plugin": "^4.6.1",
+    "@typescript-eslint/parser": "^4.6.1",
```

Comment:
Are these explicit dependencies necessary now that the project is using Foundry?

Reply:
Unfortunately yes :(

Reply:
Huh, that's weird. What breaks if you remove them?

Reply:
This one is due to the `eslint-plugin` missing, for example:

![image](https://user-images.githubusercontent.com/2780941/98804476-8a1d7980-2416-11eb-8b52-4425e58f1ea1.png)

Reply:
Since the plugin should [provide some extra rules](https://www.npmjs.com/package/@typescript-eslint/eslint-plugin#supported-rules), right?

Reply:
Ah, I think it's because Foundry [uses an older version of the plugin (v2.x)](https://github.com/sumup-oss/foundry/blob/main/package.json#L36-L37). I suspect those rules were added later.

Reply:
That would explain it, there are the only rules being left out.